apps/editing-toolkit/editing-toolkit-plugin/tags-education/src/add-tags-education-link.js

```diff
@@ -0,0 +1,37 @@
+import { recordTracksEvent } from '@automattic/calypso-analytics';
+import { useLocalizeUrl } from '@automattic/i18n-utils';
+import { ExternalLink } from '@wordpress/components';
+import { createHigherOrderComponent } from '@wordpress/compose';
+import { addFilter } from '@wordpress/hooks';
+import { __ } from '@wordpress/i18n';
+
+const trackTagsEducationLinkClick = () =>
+	recordTracksEvent( 'wpcom_block_editor_tags_education_link_click' );
+
+const addTagsEducationLink = createHigherOrderComponent( ( PostTaxonomyType ) => {
+	return ( props ) => {
+		const localizeUrl = useLocalizeUrl();
+
+		if ( props.slug !== 'post_tag' ) {
+			return <PostTaxonomyType { ...props } />;
+		}
+
+		return (
+			<>
+				<PostTaxonomyType { ...props } />
+				<ExternalLink
+					href={ localizeUrl( 'https://wordpress.com/support/posts/tags/123' ) }
```

Comment:
The call to `localizeUrl()` still doesn't seem to be working. It should be navigating users to `https://wordpress.com/es/support/entradas/etiquetas/`.

The problem is that I thought `useLocalizeUrl()` now worked without the `<LocaleProvider>` component being in the component tree. After an upgrade to the `@wordpress/i18n` package we made a change to `useLocalizeUrl()` so that it worked even if `<LocaleProvider>` wasn't anywhere in the tree. Unfortunately it doesn't work yet because the version of `@wordpress/i18n` on wpcom isn't up to date enough yet, I think?

What this means is that we either need to go back to how you were calling `localizeUrl()` directly before, but providing an explicit locale parameter (probably easier). Or adding a `<LocaleProvider>` component to the tree. This option probably doesn't make sense because the component tree is so shallow for this hook.

Either way we need to be able to provide the current locale slug. There isn't a standard way of doing this so the ETK [sends the current user locale as a server-rendered variable](https://github.com/automattic/wp-calypso/blob/4880293de50793c3cf9af3f9a158fb55019c6396/apps/editing-toolkit/editing-toolkit-plugin/wpcom-block-editor-nux/class-wpcom-block-editor-nux.php#L62). You can grab it from the `window.wpcomBlockEditorNuxLocale` global.
This adds a dependency on the `WPCOM_Block_Editor_NUX` being loaded, which is probably fine. But to make the tags-education module totally independent of wpcom-block-editor-nux you could have your own `wp_localize_script()` with your own global.

Reply:
Oops...forgot to change back the url for testing 🤣

Reply:
> The problem is that I thought useLocalizeUrl() now worked without the <LocaleProvider> component being in the component tree. After an upgrade to the @wordpress/i18n package we made a change to useLocalizeUrl() so that it worked even if <LocaleProvider> wasn't anywhere in the tree. Unfortunately it doesn't work yet because the version of @wordpress/i18n on wpcom isn't up to date enough yet, I think?

Actually, `useLocalizeUrl()` can work well even if `<LocaleProvider>` wasn't anywhere in the tree. However, there are two problems:
- Currently, `useLocalizeUrl` will try to get language from `@wordpress/i18n` under `default` domain. However, if there are any plugins with incomplete translation, it would cause the domain configuration to be overwritten so that `useLocalizeUrl` cannot get the correct language. See: D65529-code
- Besides, even if we can get the correct language, the format might not be ISO-639 and doesn't match the locale format which `localizeUrl` needs.